src/main/java/mate/controller/AddDriverForCarController.java

```diff
@@ -0,0 +1,38 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/add")
+public class AddDriverForCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final String ADD_DRIVER_FOR_CAR = "/WEB-INF/views/cars/drivers/add.jsp";
+    private CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher(ADD_DRIVER_FOR_CAR).forward(req, resp);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Car car = carService.get(Long.parseLong(req.getParameter("car_id")));
+        Driver driver = driverService.get(Long.parseLong(req.getParameter("driver_id")));
+        carService.addDriverToCar(driver, car);
+        resp.sendRedirect("/cars");
```

Comment:
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/96740815/203947029-00f22cee-eab9-463d-85cd-764cf0f7c5b9.png">
check it everywhere